python/simpler/task_interface.py

```
@@ -57,6 +59,57 @@
+    """
+    import subprocess  # noqa: PLC0415
+
+    built_from = getattr(_ti_module, "__build_commit__", "")
+    if not built_from:
+        return
+    repo_root = Path(__file__).resolve().parents[2]
+    if not (repo_root / ".git").exists():
+        return
+    try:
+        result = subprocess.run(
+            ["git", "rev-parse", "HEAD"],
```

Ruff (0.15.21)

[error] Starting a process with a partial executable path

(S607)